app/controllers/application_controller/performance.rb

```diff
@@ -426,21 +425,21 @@ def chart_current_hourly(ts)
 
     render :update do |page|
       page << javascript_prologue
-      if @parent_chart_data
-        page << 'ManageIQ.charts.chartData = ' + {
-          "candu"  => @chart_data,
-          "parent" => @parent_chart_data
-        }.to_json + ';'
-      elsif @parent_chart_data
-        page << 'ManageIQ.charts.chartData = ' + {
-          "candu"      => @chart_data,
-          "compare_vm" => @compare_vm_chart_data
-        }.to_json + ';'
-      else
-        page << 'ManageIQ.charts.chartData = ' + {
-          "candu" => @chart_data
-        }.to_json + ';'
-      end
+      page << if @parent_chart_data
+                'ManageIQ.charts.chartData = ' + {
+                  "candu"  => @chart_data,
+                  "parent" => @parent_chart_data
+                }.to_json + ';'
+              elsif @parent_chart_data
```

Comment:
is this a bug where `parent_chart_data` was copypasted here instead of `compare_vm_chart_data`?

Reply:
(added a comment in https://github.com/ManageIQ/manageiq-ui-classic/pull/652/files#r226995650 .. but .. probably this should be `elsif @compare_vm_chart_data` given line 37)

Reply:
never mind, @romanblanco is looking into this in a separate PR, merging as is